docs/Roadmap.md

```diff
@@ -1,7 +1,10 @@
-Our latest roadmap is below - please don't hesitate to feedback!
+# SpatialOS Unity GDK roadmap
+**Date:** 2018-06-19
+
+We'd love to hear what you think about the roadmap; [give us feedback](../README.MD#give-us-feedback) on what you'd like to see in it.
 
 | | Faster development iteration | Native workflows| High performance|
 |------|---|----|--|
-|First look|**Build less often**<br/>Run client and server simultaneously in your Unity Editor.<br/><br/>**Transform Feature Module**<br/>Buffer and interpolate position and rotation updates.|**Automatic synchronisation**<br/>Generated code synchronises changes to components for you with a full override control option.|**Harness the power of ECS**<br/>Take full advantage of the Unity Entity Component System (ECS).
+|First look: pre-alpha (from 2018-06-19)|**Build less often**<br/>Run client and server running simultaneously in your Unity Editor.<br/><br/>**Transform Feature Module**<br/>Buffer and interpolate position and rotation updates.|**Automatic synchronisation**<br/>Generated code synchronises changes to components for you with a full override control option.|**Harness the power of ECS**<br/>Tale full advantage of the Unity Entity Component System (ECS).
```

Comment:
Typo - "tale"

Reply:
Ooh, thanks. Not sure how they crept in there.

Reply:
Ooh, thanks. Not sure how they crept in there.